shared/wallets/onboarding/disclaimer.js

```diff
@@ -51,6 +51,7 @@ class Disclaimer extends React.Component<DisclaimerProps, DisclaimerState> {
         type="Secondary"
         onClick={this.props.onAcceptDisclaimer}
         label={label}
+        labelStyle={{color: Styles.globalColors.purple}}
```

Comment:
We should put all styles down in the `styleSheetCreate` block.

Reply:
@adamjspooner It looks like this is the only way to change the label color? See same prop used on the Open wallet button: https://github.com/keybase/client/pull/14790/files#diff-e684b085cf794d0fa3273fb86ee9f568R16

Reply:
You should be able to create a new object down in the `styleSheetCreate` block and reference it here. Something like:

```
label: {
    color: Styles.globalColors.purple,
},
```

And then `labelStyle={styles.label}` here. Does that make sense?

Reply:
got it. Updated.